examples/imagenet/imagenet_lib.py

```diff
@@ -44,56 +31,37 @@
 import jax.numpy as jnp
 
 import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
 
+import flax
+from flax import jax_utils
+from flax import nn
+from flax import optim
+from flax.metrics import tensorboard
+from flax.training import checkpoints
+from flax.training import common_utils
 
-FLAGS = flags.FLAGS
-
-flags.DEFINE_float(
-    'learning_rate', default=0.1,
-    help=('The learning rate for the momentum optimizer.'))
-
-flags.DEFINE_float(
-    'momentum', default=0.9,
-    help=('The decay rate used for the momentum optimizer.'))
-
-flags.DEFINE_integer(
-    'batch_size', default=128,
-    help=('Batch size for training.'))
-
-flags.DEFINE_bool(
-    'cache', default=False,
-    help=('If True, cache the dataset.'))
-
-flags.DEFINE_integer(
-    'num_epochs', default=90,
-    help=('Number of training epochs.'))
-
-flags.DEFINE_string(
-    'model_dir', default=None,
-    help=('Directory to store model data.'))
-
-flags.DEFINE_bool(
-    'half_precision', default=False,
-    help=('If bfloat16/float16 should be used instead of float32.'))
+import input_pipeline
+from resnet_v1 import ResNet
 
 
-def create_model(key, batch_size, image_size, model_dtype):
+def _create_model(key, batch_size, image_size, model_dtype):
```

Comment:
Let's keep these public methods.

Reply:
Done. 

Since we wanted to have type annotation for public methods with appropriate docstrings, I felt that the ones which are local/private, we could stick with the standard.